<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qmessage2_Dialog</class>
 <widget class="QDialog" name="qmessage2_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>96</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>261</width>
     <height>96</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/S4-0132.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="qmessage2_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>261</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>54</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="yes_pushButton">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 170, 0);</string>
      </property>
      <property name="text">
       <string>Yes</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="no_pushButton">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 170, 0);</string>
      </property>
      <property name="text">
       <string>No</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
